### PR TITLE
Update and publish docs for 3.3

### PR DESCRIPTION
**Pull request by mumrah**
Refresh the latest 3.3.0 docs from the 3.3 branch, and update the top-level links to point to 3.3 docs.

### CODE REVIEW

File: 33/ops.html [hunk 1]
```diff
@@ -3401,6 +3401,121 @@ <h4 class="anchor-heading"><a id="zkops" class="anchor-link"></a><a href="#zkops
     <li>Don't overbuild the cluster: large clusters, especially in a write heavy usage pattern, means a lot of intracluster communication (quorums on the writes and subsequent cluster member updates), but don't underbuild it (and risk swamping the cluster). Having more servers adds to your read capacity.</li>
   </ul>
   Overall, we try to keep the ZooKeeper system as small as will handle the load (plus standard growth capacity planning) and as simple as possible. We try not to do anything fancy with the configuration or application layout as compared to the official release as well as keep it as self contained as possible. For these reasons, we tend to skip the OS packaged versions, since it has a tendency to try to put things in the OS standard hierarchy, which can be 'messy', for want of a better way to word it.
+
```
Comment (niket-goel) [inline 33/ops.html:3404]: There were some more changes in this file from the following PR https://github.com/apache/kafka/pull/12679
Have those been pushed to the site already?

**Review by niket-goel — COMMENTED** [review]
Hey @mumrah. Thanks for the change. Does this PR represent the docs in their latest state being pushed to the asf website? If so then it is definitely missing content from one PR. In general how can we verify the changes are complete?

**Review by bbejeck — APPROVED** [review]
LGTM - Rendering locally, the quickstart still refers to 3.2, though - I don't see how as the `templateData.js` file is updated

File: 33/ops.html (hunk 1 above)
Comment (mumrah) [reply 33/ops.html:3404]: did those changes get backported to the 3.3 branch?

**Comment by bbejeck** [comment]
@mumrah thanks for the update - I clicked around on a bunch of links and from what I can see the docs point to the correct ones, including the quickstart  which points to 3.3 now

File: 33/ops.html (hunk 1 above)
Comment (niket-goel) [reply 33/ops.html:3404]: I do see it on the 3.3 branch -- https://github.com/apache/kafka/commit/2e952805431f9d4a7deef5c7f8453c63aa9702db

**Review by niket-goel — APPROVED** [review]
LGTM.

**Review by cmccabe — APPROVED** [review]

**Pull request merged**